apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ComposeView.kt

```diff
@@ -610,7 +610,17 @@ fun ComposeView(
                 if (remoteHost == null) saveAnimImage(it.uri)
                 else CryptoFile.desktopPlain(it.uri)
               is UploadContent.Video ->
-                if (remoteHost == null) saveFileFromUri(it.uri)
+                if (remoteHost == null) saveFileFromUri(
+                  it.uri,
+                  getDestFileName = { filename ->
```

Comment:
I don't like indirection here. Would be better to pass some parameter to saveFileFromUri like:
`fileNamePrefix: String?` which then be transferred to generateNewFileName (if not null) or uniqueCombine (if null)